src/app/main/component/user/components/profile/users-friends/users-friends.component.ts

```diff
@@ -72,13 +72,14 @@ export class UsersFriendsComponent implements OnInit, OnDestroy {
     this.calculateFriendsToShow();
   }
 
-  public changeFriends(isNext: boolean) {
+  changeFriends(isNext: boolean): void {
+    const totalPages: number = Math.ceil(this.amountOfFriends / this.friendsToShow);
```

Comment:
The total number of pages is returned in response from the server anyway, so why not just use that value?

Reply:
added `this.totalPages = item.totalPages;`